src/commands/artifacts_cmds/download.ts

```diff
@@ -94,7 +94,13 @@ async function handlerMain(argv: ArtifactsDownloadOptions): Promise<any> {
   const filesToDownload = argv.all
     ? artifacts.map(ar => ar.filename)
     : argv.names;
-  const nameToArtifact = _.fromPairs(artifacts.map(ar => [ar.filename, ar]));
+  const nameToArtifact = artifacts.reduce(
+    (dict, artifact) => {
+      dict[artifact.filename] = artifact;
+      return dict;
```

Comment:
Starting with Node 8.3, you can use object spread here for better readability, which might even remove the need for the type annotation below. Same in `crates/getPublishOrder`.

Reply:
Can you elaborate on how object spread would help here?

Reply:
Was thinking like this, although looking at it now it's also not much more readable :)

```js
const nameToArtifact = artifacts.reduce((dict, a) => ({...d, [a.filename]: a}), {});
```